tee-worker/scripts/litentry/release/prepare.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euo pipefail
+
+# Edit the following variable(s):
+WORKER_DIR=/tmp/worker/
+
+RUNNING_MODE_CONFIG=$1/config.json
+PRIVATE_ACCOUNT_JSON=$1/private_account.json
```

Comment:
what is this private account?

Reply:
This is the sudo account prepared for registering mrenclave.